server/gitrest/packages/gitrest-base/src/test/utils.ts

```diff
@@ -8,6 +8,11 @@ import nconf from "nconf";
 import rimrafCallback from "rimraf";
 import { IStorageDirectoryConfig } from "../utils";
 
+export interface ITestMode {
+    name: string;
+    gitLibrary: string;
```

Comment:
nit: `: "nodegit" | "isomorphic-git"`?

Reply:
I had initially done it like that, but then I thought, "maybe we will have another git library one day". It's probably not something likely to happen :) so let me update it.